stack-api/recipes/role-nginxphpapp.rb

```diff
@@ -55,6 +55,13 @@
   to '/usr/bin/php'
 end
 
+# PHP by default installs a sessioncleanup cron-job, which
+# a) we do not need, since all our session are stored in memcache
+# b) causes a PHP warning on all instances to be sent out to sysops@ due to currently missing tideways.so for php-7.1
+file '/etc/cron.d/php' do
```

Comment:
I would probably generalize this in `php-fpm::default` or `php-fpm::configure`. There are examples how to do it only if _this_ PHP is used/installed.